Client/Frontend/Widgets/AlternateSimpleHighlightCell.swift

```diff
@@ -0,0 +1,199 @@
+        selectedOverlay.backgroundColor = AlternateSimpleHighlightCellUX.SelectedOverlayColor
+        selectedOverlay.hidden = true
+        return selectedOverlay
+    }()
+
+    override var selected: Bool {
+        didSet {
+            self.selectedOverlay.hidden = !selected
+        }
+    }
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        layer.shouldRasterize = true
+        layer.rasterizationScale = UIScreen.mainScreen().scale
+
+        isAccessibilityElement = true
+
+        contentView.addSubview(siteImageView)
+        contentView.addSubview(descriptionLabel)
+        contentView.addSubview(selectedOverlay)
+        contentView.addSubview(titleLabel)
+        contentView.addSubview(timeStamp)
+        contentView.addSubview(statusIcon)
+        contentView.addSubview(domainLabel)
+
+        siteImageView.snp_remakeConstraints { make in
+            make.top.equalTo(contentView).offset(AlternateSimpleHighlightCellUX.CellTopBottomOffset)
+            make.bottom.equalTo(contentView).offset(-AlternateSimpleHighlightCellUX.CellTopBottomOffset).priorityLow()
+            make.leading.equalTo(contentView).offset(AlternateSimpleHighlightCellUX.CellSideOffset)
+            make.height.equalTo(76)
+            make.width.equalTo(99)
+        }
+
+        selectedOverlay.snp_remakeConstraints { make in
+            make.edges.equalTo(contentView)
+        }
+
+        domainLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(siteImageView.snp_trailing).offset(AlternateSimpleHighlightCellUX.CellTopBottomOffset)
+            make.top.equalTo(siteImageView).offset(-2)
+            make.bottom.equalTo(titleLabel.snp_top).offset(-4)
+        }
+
+        titleLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(siteImageView.snp_trailing).offset(AlternateSimpleHighlightCellUX.CellTopBottomOffset)
+            make.trailing.equalTo(contentView).inset(AlternateSimpleHighlightCellUX.CellSideOffset)
+        }
+
+        descriptionLabel.snp_remakeConstraints { make in
+            make.leading.equalTo(statusIcon.snp_trailing).offset(AlternateSimpleHighlightCellUX.TitleLabelOffset)
+            make.bottom.equalTo(statusIcon)
+        }
+
+        timeStamp.snp_remakeConstraints { make in
+            make.trailing.equalTo(contentView).inset(AlternateSimpleHighlightCellUX.CellSideOffset)
+            make.bottom.equalTo(descriptionLabel)
+        }
+
+        statusIcon.snp_remakeConstraints { make in
+            make.size.equalTo(SimpleHighlightCellUX.StatusIconSize)
+            make.leading.equalTo(titleLabel)
+            make.bottom.equalTo(siteImageView).priorityLow()
+            make.top.greaterThanOrEqualTo(titleLabel.snp_bottom).offset(6).priorityHigh()
+            make.bottom.lessThanOrEqualTo(contentView).offset(-AlternateSimpleHighlightCellUX.CellTopBottomOffset).priorityHigh()
```

Comment:
Nit: I feel like these constraints don't need to have special priority treatment and the priorities can be changed on the label instead since that's what stretches the height. I could be wrong though - always hard to infer from looking at autolayout code.